docker/db-migration/migrations/initial-access-management.yaml

```diff
@@ -0,0 +1,13 @@
+---
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: surabujin
+      changes:
+        - sql: "INSERT INTO ORole SET name = 'kilda-role', mode = 0, inheritedRole = (SELECT FROM ORole WHERE name = 'writer')"
```

Comment:
I'd recommend to stick to the "one change per changeset" rule. https://www.liquibase.org/get-started/best-practices

Reply:
I would like to keep meaningful transactions i.e. all required in one transaction should go in one changeset.